layers/+irc/erc/packages.el

```diff
@@ -214,7 +214,8 @@
   (use-package erc-image
     :init (with-eval-after-load 'erc (add-to-list 'erc-modules 'image))))
 
-(defun erc/init-erc-terminal-notifier ())
+(defun erc/init-erc-terminal-notifier ()
+  (use-package erc-terminal-notifier))
```

Comment:
@Andre0991 Ping @StreakyCobra (on gitter) he should be able to help you with git.